packages/contracts/contracts/Passport.sol

```diff
@@ -187,4 +194,43 @@ contract Passport is ERC721, Ownable, AccessControlEnumerable {
       return minters;
   }
 
+  /**
+    * @dev Airdrop tokens to a list of address. Restricted to admins.
+    * @param accounts The members to which token need to be airdropped.
+    */
+  function airdrop(address[] memory accounts)
+      external
+      payable
+      onlyOwner
+  {
+      require(supply >= accounts.length, "Error, number of accounts exceeds supply");
+      require(msg.value >= price*accounts.length, "Error, Airdrop costs more");
```

Comment:
this shouldn't be required. the point of the airdrop is to retroactively and frictionlessly assign membership. especially if most of the value just circulates back to the owner, that's a waste of gas

Reply:
removed payments both to owner and Cabin as discussed yesterday. We can add it back if needed later.